src/io/aviso/exception.clj

```diff
@@ -325,6 +330,17 @@
         (flush)))
     elements))
 
+(defn- clojure-frame-or-app-frame-font
```

Comment:
Perhaps this could be named `clj-frame-font`? It conflicts a little with `:clojure-frame` but might be a bit of a clearer name.